zeebe/exporter-api/src/main/java/io/camunda/zeebe/exporter/api/Exporter.java

```diff
@@ -73,4 +73,19 @@ default void close() {}
    * @param record the record to export
    */
   void export(Record<?> record);
+
+  /**
+   * Hook to perform a data purge.
+   *
+   * <p>When this method is called, the exporter <b>deletes all data</b> that has been exported so
+   * far. This method should be blocking and only return when all data has been deleted.
+   *
+   * <p>Currently there are two scenarios where this method can be called:
+   *
+   * <ol>
+   *   <li>Before a backup is restored</il>
```

Comment:
:smile: Thanks, but since we won't have the restore functionality until 8.9, I would remove this to avoid confusing people :see_no_evil: